platform/core/README

```diff
@@ -0,0 +1,71 @@
+# PolyPod platform core
+
+This is the workspace containing the logic shared across polyPod platforms: iOS and Android.
+
+## Structure:
+
+This Rust repository follows the [Cargo Workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html) structure, as follows:
+
+- [lib](lib): The top level library that composes the together the necessary functionalities.
+- [ffi](ffi): The library meant for exporting the `core` to be used in non Rust environments, such as iOS/Android/Web.
+- [crates](crates): The many small libraries that implement a specific functionality, which then are composed in [lib](lib).
+
+## Adding new functionality
+
+- Start by adding a new crate in [crates](crates).
+- If it is just a supporting crate for other crates, just integrate it with target crates.
+- If the new crate functionality needs to be exposed the platforms:
+    - Integrate it properly in [lib](lib).
+    - Update [ffi](ffi) to properly export the functionality to the platforms.
+
+## Requirements
+
+-   [Rust toolchain](https://www.rust-lang.org/tools/install). Core is built with Rust 1.63.0.
```

Comment:
pinned? At least that one?

Reply:
What do you mean?

Reply:
I mean that it might be a good idea to specify if you mean it can be built *only* with that version, or the user will need *at least* that version.